scipy/spatial/transform/_rigid_transformation.pyx

```diff
@@ -846,6 +846,20 @@ cdef class RigidTransformation:
         This implements the exponential map that converts 6-dimensional real
         vectors to SE(3).
 
+        An exponential coordinate vector consists of 6 elements - 3 for rotation (known
+        as a rotation vector) and 3 for translation. The exponential mapping can be
+        expressed as matrix exponential ``M = exp(m)``. Where ``M`` is a 4x4 matrix
+        representing a ridig transformation and ``m`` is a 4x4 matrix formed from
```

Comment:
Instead of using the symbol $m$, I would then use $\left[\boldsymbol{\tau}\right]$ for this matrix.

Reply:
I think `tau` will do in the text format as well.

Reply:
Ok for me